foundation/foundation-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -65,6 +65,7 @@
 				<include>devops/**</include>
 				<include>jenkinsPipelineSteps.groovy</include>
 				<include>.mvn/**</include>
+				<include>Tiltfile-config</include>
```

Comment:
C: Let's not introduce this here. I don't think this is what we want the final user experience for the config store to be. So for now, we can use this approach to test as we're developing but before we ship we'll need a better solution.